### PR TITLE
ci: skip slash-command-action if comment does not match

### PR DESCRIPTION
**Pull request by YangKian**
Currently, `xt0rted/slash-command-action` will fail when a non-matching comment is created. Refer to https://github.com/xt0rted/slash-command-action/issues/124. This pr skips the entire job if the comment does not match.

### CODE REVIEW

File: .github/workflows/issue-cmds.yml
Comment (drmingdrmer) [inline .github/workflows/issue-cmds.yml:10]: I do not quite get it: 
Does `github.event.issue.pull_request` mean it has to be a `pull_request` to execute the following steps?
These actions are meant to serve for issues not PRs.

And I do not know yet what you actually encountered. What is it that does not work as expected?

Comment (YangKian) [reply .github/workflows/issue-cmds.yml:10]: Sorry, I should describe the question more clear. See https://github.com/datafuselabs/openraft/actions/runs/1653115173 as an example, any comment that doesn't match the `/assignme` and `/help` command will cause CI to fail. This pr try to fix the problem.

> Does github.event.issue.pull_request mean it has to be a pull_request to execute the following steps? These actions are meant to serve for issues not PRs.

You are right, I make a mistake here, I'll fix it soon.

Comment (drmingdrmer) [reply .github/workflows/issue-cmds.yml:10]: > any comment that doesn't match the /assignme and /help command will cause CI to fail. 

I can't confirm this: The CI includes only a unit test job which you can find it here: https://github.com/datafuselabs/openraft/blob/main/.github/workflows/ci.yaml

This action fails on every comment but should not block a PR CI IMO.
Is there a PR that's blocked by this action?

After all, filtering off non-command would make github comfortable :DDD

Comment (YangKian) [reply .github/workflows/issue-cmds.yml:10]: > Is there a PR that's blocked by this action?

No, it's just that "slash-command-action" failed, I should have used "action failed" instead of "CI failed", sorry for my misrepresentation. =-=

**Comment by YangKian** [comment]
PTAL~

**Comment by YangKian** [comment]
seems, it still does not work =-=, I'll convert it to draft and do more check

**Comment by drmingdrmer** [comment]
`client_reads` case is flaky. Re-run-ed it.

**Comment by YangKian** [comment]
test

**Comment by YangKian** [comment]
test

**Comment by YangKian** [comment]
I was stupid, I thought the current pr would take effect immediately after modifying the `issue-cmds.yml` file, but it seems that it will take effect only after the merge. Sorry this is my first time dealing with GitHub action-related issues. PTAL~

**Comment by drmingdrmer** [comment]
> I was stupid, I thought the current pr would take effect immediately after modifying the `issue-cmds.yml` file, but it seems that it will take effect only after the merge. Sorry this is my first time dealing with GitHub action-related issues. PTAL~

It is my second or third time too:DDD

**Review by drmingdrmer — APPROVED** [review]
Love it :DDD

**Comment by YangKian** [comment]
> Love it :DDD

Thanks for your patience XD

**Comment by drmingdrmer** [comment]
> > Love it :DDD
> 
> Thanks for your patience XD

And thank you for your hard work that make thing better :DDD

**Pull request merged**
